app/assets/javascripts/challenges.js

```diff
@@ -1,2 +1,12 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+
+// Purpose: Provides show and hide feature to a clickable element with "toggler" class
+// Date: 6/16/2020
+
+$(document).ready(function() {
```

Comment:
Turbolinks has a special document.ready handler that I believe had to be included here as well, if you look up turbolinks you should be able to find it.